src/main/java/com/github/rodionovsasha/commentservice/controllers/ExceptionHandlerController.java

```diff
@@ -13,7 +14,7 @@
 
 @ControllerAdvice
 public class ExceptionHandlerController {
-    @ExceptionHandler(UserNotFoundException.class)
+    @ExceptionHandler({UserNotFoundException.class, TopicNotFoundException.class})
```

Comment:
Should we inherit `UserNotFoundException` and `TopicNotFoundException` exceptions from the same parent class/interface? So here we can catch this parent class/interface instead of enumerating every "not found" exception separately